apps/public-reference/pages/applications/contact/address.tsx

```diff
@@ -125,6 +130,7 @@ export default () => {
               name="applicant.phoneNumberType"
               placeholder={t("application.contact.phoneNumberTypes.prompt")}
               label={t("application.contact.phoneNumberTypes.prompt")}
+              labelClassName={"sr-only"}
```

Comment:
Totally pedantic, but just calling it out for perhaps cleaner code - you only need brackets `{..}` when you're evaluating JS - if it's just a basic string literal, JSX lets you do `labelClassName="sr-only"`. Similarly, if it's just a boolean flag, you can do just do `caps` (rather than `caps={true}`)

Reply:
yup, agree, I'll clean up it when I'll detect more places like this :)